policy/config/output/lsstComCamDefault.yaml

```diff
@@ -0,0 +1,47 @@
+output:
+  type: LSST_CCD
+  nproc: 8    # Change this to work on multiple CCDs at once.
+  # nfiles: 1   # test
+  nfiles: 9 # Comcam
+
+  header:
+    mjd: *mjd
+    seqnum: *seqnum
+
+  camera: LsstComCam
+
+  exptime: $exptime
+
+  cosmic_ray_rate: 0.0
+
+  det_num:
+    type: List
+    items: $list(range(9))
+
+
+  file_name:
+    type: FormattedStr
+    format : raw_%s-%1d-%s-%s-det%03d.fits.fz
+    items:
```

Comment:
so to offset the piston one can use `input.telescope.focusZ`, and then appropriately set `output.header.focusZ` , did you consider changing the formatting of the output  files so that each defocus in case of `lsstComCam` produces distinguishable filename?  How else do they not clash?

Reply:
Or do you change `obsid` for each defocus?

Reply:
I ask b/c when it comes to `opsim` integration, it would be more natural to use `opsim` `visit` as `obsid` , rather than changing `obsid` for each defocal offset.

Reply:
The closed loop changes the sequence number for each successive pistoned image. https://github.com/lsst-ts/ts_imsim/blob/7184a8879d971fc28452d64ed87a4a2b92d88c2b/python/lsst/ts/imsim/closed_loop_task.py#L510

Reply:
This just uses the functionality already created and being used for FAM in the closed loop.